TripHouse/index.html

```diff
@@ -0,0 +1,36 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link href="../styles/style.css" rel="stylesheet" />
+    <link rel="preconnect" href="https://fonts.googleapis.com" />
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin />
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500&display=swap" rel="stylesheet"/>
+    <title>What do we offer</title>
+</head>
+<body>
+<div class="container-one">
+    <div class="text">What do we offer</div>
+</div>
+<hr />
+<div class="container-two">
```

Comment:
**Semantic recomendation**
It's better to use tags` <ul>` and `<li>` here, because this is a list of elements